py-polars/polars/series/binary.py

```diff
@@ -122,6 +127,18 @@ def decode(self, encoding: TransferEncoding, *, strict: bool = True) -> Series:
             b"\xff\xff\x00"
             b"\x00\x00\xff"
         ]
+
+        Mask results for invalid values with null values versus raising an error.
```

Comment:
```suggestion
        Set `strict=False` to set invalid values to null instead of raising an error.
```

Reply:
Thank you - much more clear.  Will change line as suggested.